js/toolbar.js

```diff
@@ -303,11 +303,33 @@ class Toolbar {
     renderPlayIcon(onclick) {
         const playIcon = docById("play");
         const stopIcon = docById("stop");
+        
+        let isPlayIconRunning = false;
+        function handleClick(){
+            if (!isPlayIconRunning) {
+                playIcon.onclick = 'null'
+                console.log("Wait for next 5 seconds to play the music");
+                
+            } else {
+                playIcon.onclick = tempclick;
+                isPlayIconRunning = false;
+            }
+        }    
 
-        playIcon.onclick = () => {
+        var tempclick = playIcon.onclick=()=>{
+            isPlayIconRunning = false;
             onclick(this.activity);
+            handleClick();
             stopIcon.style.color = this.stopIconColorWhenPlaying;
-        };
+            isPlayIconRunning = true; 
+            variable= setTimeout(function() { handleClick(); }, 4600);
```

Comment:
space before =

Reply:
4600 ms is too long for the timeout. Where did that come from? Otherwise, looks good except for the couple of comments above.

Reply:
it take arround little more than 4.6 seconds to run the default music block. So its the maximum time i can put it so that user don't feels any lag. may be we can reduce it . It was just for testing.

Reply:
Should I lower the timeout to 1 or 2 seconds?

Reply:
Let's try living with 2 seconds for a bit and see how it feels.